pg_search/src/postgres/customscan/opexpr.rs

```diff
@@ -66,4 +152,15 @@ impl OpExpr {
             OpExpr::Single(expr) => (*(*expr)).location,
         }
     }
+
+    pub fn is_text(&self) -> bool {
```

Comment:
This could maybe use a better name. It's not clear what "is_text" means in the context of an operator: does the mean it requires text on the LHS and RHS, or that it accepts text on one side or the other, etc.